### PR TITLE
Feature/secret import fix

### PR DESCRIPTION
**Pull request by haitiangk92**

### CODE REVIEW

**Comment by haitiangk92** [comment]
Please reference this dag as successful runs with containers using the new feature: https://github.com/colpal/dataEng-container-tools/pull/171/files

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="dataEng-container-tools",
-    version="0.6.1",
+    version="0.6.4",
     author="Alexander Saff",
     author_email="alexander_saff@colpal.com",
```
Comment (GoodToast) [inline setup.py:11]: should we update this? Alex has not worked at Colgate for some time. what do you think @mourtallah  ?

Comment (GoodToast) [reply setup.py:11]: just from a support perspective, might make sense to add DE google group or something instead of Alex's email

Comment (matthewhenderson15) [reply setup.py:11]: I was thinking the same thing for the PKG-INFO as well, maybe also changing the name to the DE team.

Comment (haitiangk92) [reply setup.py:11]: i can see that. whats the email?

**Review by matthewhenderson15 — APPROVED** [review]
Other than the email/name changes LGTM

File: dataEng_container_tools.egg-info/PKG-INFO [hunk 2]
```diff
@@ -1,239 +1,242 @@
 Metadata-Version: 2.1
 Name: dataEng-container-tools
-Version: 0.6.1
+Version: 0.6.4
 Summary: A package containing tools for data engineering containers.
 Home-page: https://github.com/colpal/dataEng-container-tools
 Author: Alexander Saff
 Author-email: alexander_saff@colpal.com
```
Comment (GoodToast) [inline dataEng_container_tools.egg-info/PKG-INFO:7]: can we add a `Maintainer-email` field and include DE team google group?

File: setup.py (hunk 1 above)
Comment (haitiangk92) [reply setup.py]: I've updated the contact information

File: dataEng_container_tools.egg-info/PKG-INFO (hunk 2 above)
Comment (haitiangk92) [reply dataEng_container_tools.egg-info/PKG-INFO]: > hmm is there no way to sync `PKG-INFO` with the info under `setup.py`?

they should be synced. where do you see discrepancies?

**Review by mourtallah — APPROVED** [review]

**Review by robbiedow — APPROVED** [review]

File: dataEng_container_tools.egg-info/PKG-INFO (hunk 2 above)
Comment (haitiangk92) [reply dataEng_container_tools.egg-info/PKG-INFO]: > I think PKG-INFO, being an egg file, is generated off of setup.py. I would think we can remove it.

@mourtallah What information in the PKG-INFO do you think shouldn't be in the repo?

Comment (haitiangk92) [reply dataEng_container_tools.egg-info/PKG-INFO]: > can we add a `Maintainer-email` field and include DE team google group?

@GoodToast I've added the maintainer email

**Review by GoodToast — APPROVED** [review]

**Review by matthewhenderson15 — APPROVED** [review]

**Review by mourtallah — APPROVED** [review]

**Pull request merged**